pynestml/codegeneration/nest_builder.py

```diff
@@ -85,14 +85,15 @@ def __init__(self, options: Optional[Mapping[str, Any]] = None):
             try:
                 import nest
             except ModuleNotFoundError:
-                Logger.log_message(None, -1, "An error occurred while importing the `nest` module in Python. Please check your NEST installation-related environment variables and paths.", None, LoggingLevel.ERROR)
+                Logger.log_message(
+                    None, -1, "An error occurred while importing the `nest` module in Python. Please check your NEST installation-related environment variables and paths.", None, LoggingLevel.ERROR)
                 sys.exit(1)
 
             nest_path = nest.ll_api.sli_func("statusdict/prefix ::")
             self.set_options({"nest_path": nest_path})
             Logger.log_message(None, -1, "The NEST installation was automatically detected as: " + nest_path, None, LoggingLevel.INFO)
 
-    def build(self) -> None:
+    def build(self, stdout: TextIO = None, stderr: TextIO = None) -> None:
```

Comment:
Could you add docstrings for these new parameters and perhaps a little instruction on how they can be used?